tests/conftest.py

```diff
@@ -176,14 +175,12 @@ def jaxsim_model_ergocub() -> js.model.JaxSimModel:
         The JaxSim model of the ErgoCub robot.
     """
 
-    os_environ_original = os.environ.copy()
-
     try:
         os.environ["ROBOT_DESCRIPTION_COMMIT"] = "v0.7.1"
 
         import robot_descriptions.ergocub_description
     finally:
-        os.environ = os_environ_original
+        os.environ.clear()
```

Comment:
Doesn't `clear()` delete the entire dictionary? Here I want to restore the system one.

Reply:
`clear()` deletes the environment variable defined during the process started by Python, we could eventually think to use `subprocess.Popen` to pass a custom env keys set to a subprocess

Reply:
It doesn't seem that it works as you described it:

```
❯ ipython
Python 3.11.8 | packaged by conda-forge | (main, Feb 16 2024, 20:53:32) [GCC 12.3.0]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.22.2 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import os

In [2]: os.environ["HOME"]
Out[2]: '/home/dferigo'

In [3]: os.environ["FOO"] = "bar"

In [4]: os.environ.clear()

In [5]: os.environ["HOME"]
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
Cell In[5], line 1
----> 1 os.environ["HOME"]

File <frozen os>:679, in __getitem__(self, key)

KeyError: 'HOME'
```

What's the problem of the previous logic?

Reply:
Ow ok now I understand the problem. What about the following logic?

```python
    try:
        os.environ["ROBOT_DESCRIPTION_COMMIT"] = "v0.7.1"

        import robot_descriptions.ergocub_description
    finally:
        _ = os.environ.pop("ROBOT_DESCRIPTION_COMMIT", None)
```

Reply:
Sure, it makes sense

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/114/commits/4cd9358b897b1efdd9621b0c32d31897920a0655